util-node/apiDocsTransformUtils.js

```diff
@@ -131,9 +131,7 @@ function createApiDocsFrontmatter(firstLine, { version, name }) {
 // into a proper metadata with the information (index number) and any other accompanying text
 function replaceStabilityIndex(metadata) {
   return (_, __, level, text) => {
-    const sanitizedText = text
-      .replace(/(\n)>/g, '')
-      .replace(/\[(.+)\]\[\]/g, (___, piece) => piece);
+    const sanitizedText = text.replace(/(\n)>/g, '');
```

Comment:
This doesn't fix the links issue, it only makes the Markdown syntax to be present there, but they're not converted into links.